libraries/Microsoft.Bot.Builder.AI.QnA/Utils/Constants.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.AI.QnA
```

Comment:
Why is this public class in the Utils folder? 
Consider moving it to the Models or other more meaningful folder and update the namespace to match the location